samples/universal-react-node/package.json

```diff
@@ -46,6 +46,7 @@
     "electrode-archetype-react-app-dev": "^1.6.0",
     "redux-devtools": "^3.3.1",
     "redux-devtools-dock-monitor": "^1.1.1",
-    "redux-devtools-log-monitor": "^1.0.11"
+    "redux-devtools-log-monitor": "^1.0.11",
+    "superagent": "^3.3.1"
```

Comment:
should use isomorphic-fetch polyfill so if browser supports it natively then it can be used.